management-api-agent-common/src/main/java/com/datastax/mgmtapi/NodeOpsProvider.java

```diff
@@ -940,6 +941,29 @@ public String move(
     return submitJob("move", moveOperation, async);
   }
 
+  @Rpc(name = "reloadInternodeEncryptionTruststore")
+  public void reloadInternodeEncryptionTruststore() throws Exception {
+    ShimLoader.instance.get().reloadInternodeEncryptionTruststore();
+  }
+
+  @Rpc(name = "getInternodeEncryptionTruststoreIssuers")
+  public List<String> getEncryptionTruststoreIssuers() throws Exception {
```

Comment:
question Why is this feature required for CA rotation?

Reply:
It will be required for e2e testing this inside this repo. We may not do that just yet (depending on time constraints) but if we don't include it there is no way to introspect the contents of the truststore as Cassandra sees them.

Reply:
This repo is not meant to test DSE features though, if you use the same API as nodetool, then we can assume it just works. 

Also, since we didn't want to include client truststore information, we shouldn't add anything else that isn't used right now and not related to CA rotation.

There's other issue with it exposing also the truststore content outside the security realm of how Secrets are handled.